packages/expo-passkeys/package.json

```diff
@@ -0,0 +1,47 @@
+{
+  "name": "@clerk/expo-passkeys",
+  "version": "0.0.1",
+  "description": "Passkeys library to be used with Clerk for expo",
+  "keywords": [
+    "react-native",
+    "expo",
+    "expo-passkey",
+    "ClerkExpoPasskeys",
+    "clerk",
+    "passkeys"
+  ],
+  "homepage": "https://clerk.com/",
+  "bugs": {
+    "url": "https://github.com/clerk/javascript/issues"
+  },
+  "repository": "git+https://github.com/clerk/javascript.git",
+  "license": "MIT",
+  "author": "Clerk",
+  "main": "build/index.js",
+  "types": "build/index.d.ts",
+  "scripts": {
+    "build": "EXPO_NONINTERACTIVE=1 expo-module build",
+    "build:watch": "expo-module build",
+    "clean": "expo-module clean",
+    "expo-module": "expo-module",
+    "lint": "expo-module lint",
+    "open:android": "open -a \"Android Studio\" example/android",
+    "open:ios": "xed example/ios",
+    "prepare": "expo-module prepare",
+    "prepublishOnly": "expo-module prepublishOnly"
+  },
+  "dependencies": {
+    "@clerk/shared": "2.10.1-snapshot.va96a299",
+    "@clerk/types": "4.28.0-snapshot.va96a299"
```

Comment:
Shouldn't this be the latest version and not a snapshot version?

Reply:
Yes but I will address update the version of clerk in this [PR](https://github.com/clerk/javascript/pull/4352), the reason is we depend on some types that I exported there.